pkg/lang/ir/install-mamba.sh

```diff
@@ -0,0 +1,14 @@
+set -euo pipefail && \
+ARCH="$(uname -m)" && \
+if [[ "${ARCH}" == "aarch64" ]]; then \
+    ARCH="aarch64"; \
+elif [[ "${ARCH}" == "ppc64le" ]]; then \
+    ARCH="ppc64le"; \
+else \
+    ARCH="64"; \
+fi && \
+mkdir -p ${MAMBA_BIN_DIR} && \
+curl -Ls https://micro.mamba.pm/api/micromamba/linux-${ARCH}/latest | tar -xvj -C ${MAMBA_BIN_DIR} --strip-components=1 bin/micromamba && \
+ln -s ${MAMBA_BIN_DIR}/micromamba ${MAMBA_BIN_DIR}/conda && \
+echo -e "channels:\n  - conda-forge" > ${MAMBA_ROOT_PREFIX}/.mambarc
```

Comment:
`conda-forge` is used by default.

This will install `numpy`, `scipy`, etc. with OpenBLAS. Check the doc here: https://numpy.org/install/#numpy-packages--accelerated-linear-algebra-libraries

Reply:
Does `condig.conda_channel` work for mamba?

Reply:
Yes. Check https://mamba.readthedocs.io/en/latest/user_guide/configuration.html#multiple-rc-files